src/sentry/models/identity.py

```diff
@@ -36,11 +37,43 @@ class IdentityProvider(Model):
     """
 
     __include_in_export__ = False
+    organization = FlexibleForeignKey("sentry.Organization", blank=True, null=True)
 
+    # To be renamed to "provider" because this is a bad column name
     type = models.CharField(max_length=64)
+    # Replacing "type"
+    provider = models.CharField(max_length=64, null=True)
+
+    # To be renamed to "provider_id", because we're already renaming "type"
+    external_id = models.CharField(max_length=64, null=True)
+    # Replacing "external_id"
+    provider_id = models.CharField(max_length=64, null=True)
     config = EncryptedJsonField()
+
     date_added = models.DateTimeField(default=timezone.now, null=True)
-    external_id = models.CharField(max_length=64, null=True)
+
+    # SSO-specific fields
+    is_sso = models.BooleanField(default=False)
+    sso_default_team = models.OneToOneField(
```

Comment:
This was `ManyToMany` on `AuthProvider` but we don't use it at all. 

I don't see a use-case for declaring multiple default teams for a single provider.